apps/src/pythonlab/pythonlab-view.module.scss

```diff
@@ -1,15 +1,4 @@
-$default-spacing: 2px;
-
```

Comment:
super nit: Could update style rule name to `.pythonlabContainer` similar to naming of `weblab2Container`.